packages/core/src/components/helpcenter-panel/helpcenter-panel.scss

```diff
@@ -27,20 +26,20 @@
 }
 
 .dimscreen {
-  background-color: $zwart;
+  background-color: colors.$zwart;
   height: 100%;
   opacity: 0.4;
   width: 100%;
 }
 
 .open-button {
-  @include button_primary($modifiers: false);
+  @include shared-button-mixins.button_primary($modifiers: false);
```

Comment:
`button_primary()` moet `primary()` worden, idem voor secondary en tertiary

Reply:
Ik heb alle andere button smaken (`button_small, etc`) ook mee genomen

Reply:
Goed punt. De `_` notatie is onze eigen pre-Sass modules namespace conventie. Het namespacen wordt nu door Sass modules gerealiseerd en moet dus worden ontmanteld.